app/modules/intelligence/provider/provider_controller.py

```
@@ -15,7 +16,7 @@ def __init__(self, db: Session, user_id: str):
     async def list_available_llms(self) -> List[ProviderInfo]:
         try:
             providers = await self.service.list_available_llms()
```

Ruff (0.8.2)

Local variable `providers` is assigned to but never used

Remove assignment to unused variable `providers`

(F841)